src/Collection/Hero.php

```diff
@@ -185,4 +266,36 @@ public static function init( $locations ) {
 
 		endif;
 	}
+
+	/**
+	 * Format the background field.
+	 *
+	 * @param array $value The value.
+	 * @param int $post_id The post id.
+	 * @param array $field The acf field definition.
+	 * @return mixed
+	 */
+	public static function format_background( $value, $post_id, $field ) {
+		if ( self::BACKGROUND !== $field['key'] ) {
+			return $value;
+		}
+
+		foreach ( $value as $index => $item ) {
```

Comment:
Ok why not construct an array with the relevant information only.

- Removes code that does not give any value (all the lines with unset)
- Makes function short
- Improves readability and keep things simple.